umbraco-cloud/release-notes/2024-05-releasenotes.md

```diff
@@ -0,0 +1,23 @@
+# May 2024
+
+## Key Takeaways
+
+* **Upload Custom Database** - You can now upload your Umbraco database onto the Umbraco Cloud. (Add additional text, restoring is right around the corner!)
+
+* **Dedicated Environments** - You can now decide how to best host your Umbraco website with dedicated Environments, with the option to isolate your Live environment on a dedicated server.
+
+
+## Upload Custom Database
+
+Introduction the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling backing up and restoring Umbraco Cloud websites. 
```

Comment:
```suggestion
With the introduction of the Point-In-Time Recovery (PITR) feature on the Backups page, we started the process of enabling back up and restore capabilities for Umbraco Cloud websites.. 
```

Reply:
THe sentence will be too long with your suggestion 🙈

I have tried to add another suggestion.